### PR TITLE
Bump dependencies to stable and add RTL styles

### PR DESCRIPTION
**Pull request by yuriy-fix**

### CODE REVIEW

**Comment by yuriy-fix** [comment]
@diego, I have updated the screenshots, but I am not sure that it is the correct behavior in RTL with icon position switched. It looks like the value is clipped now

**Comment by DiegoCardoso** [comment]
Yeah. That's coming from the `vaadin-date-picker` where we use `direction: ltr` on the `[part=input-field]`. It seems a bit off, I agree. In one hand it is consistent with the icon being opposite to the text. But that style also implies that the suffix and prefix for RTL are kept on the same position as in LTR.

Should we revert the `flow-direction` as well, so prefix would be on the right and suffix on the left of the input? (If we do so, maybe we should apply the same all email-field, number-field and decimal-field)

**Comment by Haprog** [comment]
Look like at least border radius is now missing from one of the elements in the middle section (between elements) in both LTR and RTL.

**Comment by DiegoCardoso** [comment]
Indeed. I had to zoom-in to catch that difference.

**Comment by DiegoCardoso** [comment]
Found the issue with the borders. Because now `vaadin-date-picker` creates its own `vaadin-date-picker-text-field` instead of plain `vaadin-text-field` this line needs to be updated: https://github.com/vaadin/vaadin-date-time-picker/blob/master/src/vaadin-date-time-picker-date-picker.html#L37

File: src/vaadin-date-time-picker-date-text-field.html [hunk 1]
```diff
@@ -6,6 +6,32 @@
 
 <link rel="import" href="../../vaadin-text-field/src/vaadin-text-field.html">
 
+<dom-module id="vaadin-date-time-picker-date-text-field-styless" theme-for="vaadin-date-time-picker-date-text-field">
+  <template>
+    <style>
+      :host([dir="rtl"]) [part="input-field"] {
```
Comment (Haprog) [inline src/vaadin-date-time-picker-date-text-field.html:12]: Wouldn't it make more sense to import `<link rel="import" href="../../vaadin-date-picker/src/vaadin-date-picker-text-field.html">` and then something like:
```js
class DateTimePickerDateTextFieldElement extends customElements.get('vaadin-date-picker-text-field')
```
Then we wouldn't need to duplicate the RTL styles here.

File: theme/lumo/vaadin-date-time-picker-styles.html [hunk 2]
```diff
@@ -16,6 +16,17 @@
...
+      :not(*):placeholder-shown, /* to prevent broken styles on IE */
+      :host([dir="rtl"]) [part="value"]:placeholder-shown,
+      :host([dir="rtl"]) [part="input-field"] ::slotted(input:placeholder-shown) {
+        --_lumo-text-field-overflow-mask-image: none;
+      }
+
+      :host([dir="rtl"]) [part="value"],
+      :host([dir="rtl"]) [part="input-field"] ::slotted(input) {
+        --_lumo-text-field-overflow-mask-image: linear-gradient(to left, transparent, #000 1.25em);
+      }
```
Comment (Haprog) [inline theme/lumo/vaadin-date-time-picker-styles.html:29]: I guess these can also be removed from here if this element is made to extend `vaadin-date-picker-text-field`?

File: src/vaadin-date-time-picker-date-text-field.html (hunk 1 above)
Comment (Haprog) [reply src/vaadin-date-time-picker-date-text-field.html:12]: Also noticed there's a typo in `-styless"`

**Pull request merged**